### ISSUE TITLE
GraphQL Page API - Implement `render` field at Page level 

### ISSUE THREAD

**Issue by dsilvam** [dotCMS/core#19143]
```
{
  page(url:"/destinations/costa-rica", pageMode: "live", languageId: "1") {
     render
  }
}
```
Should return the rendered `html` of the given page

**Comment by dsilvam** [comment]
PR: https://github.com/dotCMS/core/pull/19144

**Comment by freddyucv** [comment]
**Passed Internal QA**: Check postman test

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF
